site/en/articles/css-linear-easing-function/index.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Create complex animation curves in CSS with the `linear()` easing function
+authors:
+  - bramus
+subhead: >
+  Introducing `linear()`, an easing function in CSS that interpolates linearly between its points, allowing you to recreate bounce and spring effects.
+description: >
+  Introducing `linear()`, an easing function in CSS that interpolates linearly between its points, allowing you to recreate bounce and spring effects.
+date: 2023-05-25
+hero: "image/AeNB0cHNDkYPUYzDuv8gInYA9rY2/tlphA8yFApB07mM9Oxe2.jpg"
+alt: "Birds in the sky, forming a line as they fly."
+tags:
+  - css
+  - chrome-113
+---
+
+## Easings in CSS
+
+When animating or transitioning elements in CSS, you control the rate at which a value changes with an easing function using the `animation-timing-function` and `transition-timing-function` properties.
+
+There are several keywords available as presets in CSS, namely `linear`, `ease`, `ease-in`, `ease-out`, and `ease-in-out`. To create your own easing curves use the `cubic-bezier()` function, or  take a steps based approach using the `steps()` easing function.
+
+{% Codepen {
```

Comment:
there is a type in this codepen "animaton" rather than "animation" in the title.

Reply:
Heh, it’s one from the Learn CSS course and totally hadn’t noticed it myself. Fixed the pen.